src/compositor/matting/mediapipe.ts

```
@@ -0,0 +1,129 @@
+ * it. Trust-equivalent to the `importScripts(loaderUrl)` MediaPipe itself
+ * uses in classic workers. Main thread is untouched (script-tag path works).
+ */
+let loaderReady: Promise<void> | null = null;
+function ensureWasmLoaderGlobal(): Promise<void> {
+  if (typeof document !== 'undefined') return Promise.resolve();
+  const g = globalThis as { ModuleFactory?: unknown };
+  if (g.ModuleFactory) return Promise.resolve();
+  loaderReady ??= fetch(wasmLoaderUrl)
+    .then((r) => r.text())
+    .then((src) => {
+      (0, eval)(src);
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)